cookbooks/bcpc-hadoop/templates/default/hdp_hadoop-env.sh.erb

```diff
@@ -1,31 +1,55 @@
-export TEZ_CONF_DIR=/etc/tez/conf
-export TEZ_JARS=/usr/hdp/current/tez-client/*:/usr/hdp/current/tez-client/lib/*
-
+# Set Hadoop-specific environment variables here.
+ 
+# The java implementation to use.  Required.
 export JAVA_HOME=<%= node["bcpc"]["hadoop"]["java"] %>
+export HADOOP_HOME_WARN_SUPPRESS=1
+ 
+# Extra Java runtime options.  Empty by default.
+export HADOOP_OPTS="-Djava.net.preferIPv4Stack=true ${HADOOP_OPTS}"
+ 
+# Where log files are stored.  $HADOOP_HOME/logs by default.
+export HADOOP_LOG_DIR=/var/log/hadoop-hdfs
+ 
+# History server logs
+export HADOOP_MAPRED_LOG_DIR=/var/log/hadoop-hdfs
+ 
+# Where log files are stored in the secure data environment.
+export HADOOP_SECURE_DN_LOG_DIR=/var/log/hadoop-hdfs
+ 
+# The directory where pid files are stored.
+export HADOOP_PID_DIR=/var/run/hadoop-hdfs
+export HADOOP_SECURE_DN_PID_DIR=/var/run/hadoop-hdfs
+ 
+# History server pid
+export HADOOP_MAPRED_PID_DIR=/var/run/hadoop-hdfs
+ 
+# A string representing this instance of hadoop.
+export HADOOP_IDENT_STRING="$USER"
+ 
+# Use libraries from standard classpath
+JAVA_JDBC_LIBS=""
+#Add libraries required by mysql connector
+for jarFile in `ls /usr/share/java/*mysql* 2>/dev/null`
+do
+  JAVA_JDBC_LIBS=${JAVA_JDBC_LIBS}:$jarFile
+done
+#Add libraries required by oracle connector
+for jarFile in `ls /usr/share/java/*ojdbc* 2>/dev/null`
+do
+  JAVA_JDBC_LIBS=${JAVA_JDBC_LIBS}:$jarFile
+done
+#Add libraries required by nodemanager
+TEZ_LIBS=/etc/tez/conf:/usr/hdp/${HDP_VERSION}/tez/*:/usr/hdp/${HDP_VERSION}/tez/lib/*
+export HADOOP_CLASSPATH=${HADOOP_CLASSPATH}${JAVA_JDBC_LIBS}:${TEZ_LIBS}
+ 
+# Path to jsvc required by secure HDP 2.0 datanode
```

Comment:
This may be required for HDP 2.0 datanodes but we run 2.0+ (i.e. 2.2 datanodes these days...)